Do not output unknown error msg if label is built in

juan-fdz-hawa opened the pull request:
<!-- Add the related story/sub-task/bug number, like Resolves #123, or remove if NA -->
**Related issue:** Resolves #32776 

When validating labels on gitops, if the label is built-in, do not display "Please create the missing labels..." error message.

## Testing

- [X] QA'd all new/changed functionality manually

sgress454 commented on cmd/fleetctl/fleetctl/gitops.go:
```diff
@@ -260,27 +260,21 @@ func gitopsCommand() *cli.Command {
 				}
-				if unknownLabelsUsed {
-					return errors.New("Please create the missing labels, or update your settings to not refer to these labels.")
-				}
```
I think this was mostly fine thew way it was, just maybe move the `unknownLabelsUsed = true` inside the `else` so that we only get the "please create missing labels" message if there are actually missing labels?  If we return early then we don't get the whole list of missing labels, which makes debugging easier.

juan-fdz-hawa committed:
Do not output unknown error msg if label is built in

When validating labels on gitops, if the label is built-in, do not
display "Please create the missing labels..." error message.
